db/seeds.rb

```diff
@@ -1,26 +1,26 @@
+Candidate.create(name: "Joe Biden", party: "Democratic", bio: "While not officially running, Joe Biden's ambitions to show America 'where the party's at' are clear. Born to humble bartenders, Biden quickly endeared himself to the local patrons of 'Shotgun Joe's. His social acumen prompted a drunken patron one day to suggest that he run for public office. Biden, with a Cheshire Cat smile, shook off his 11am buzz, and stumbled into the office of the Vice President. Using similar tactics, he looks to repeat this success in the 2016 election.", last_name: "biden")
 puts "Candidates created"
 Issue.create(topic: "Marriage Equality", description: "The central argument in the upcoming debates will be quite simple. Kim Davis vs. humans marrying sheep. Discuss.", picture: 'marriage')
 
-Issue.create(topic: "Environment", description: "Set to be a hotly contested issue, candidates will debate whether climate change is a hoax or if the world is set to end in 20 years time.", featured: true, picture: 'environment')
+Issue.create(topic: "Environment", description: "Set to be a hotly contested issue, candidates will debate whether climate change is a hoax or if the world is set to end in 20 years time. It still remains a mystery.", featured: true, picture: 'environment')
 
-Issue.create(topic: "Immigration", description: "Some candidates insist that their families materialized from America's soil, never to have experienced immigration, while others would like to open Lady Liberty's arms to the masses.", featured: true, picture: 'immigration')
+Issue.create(topic: "Immigration", description: "Some candidates insist their families materialized from America's soil, never to have experienced immigration, while others would like to open Lady Liberty's arms to the masses.", featured: true, picture: 'immigration')
 
-Issue.create(topic: "ISIS", description: "We can already hear the arguments unfolding and the debates haven't started. Republican: 'In conjunction with sharing our deep held values of A-MUR-I-CA, cheeseburgers included, we are making great progress against ISIS with every drone strike. They're the best!' Democrats: 'No no no, drone attacks are the work of the devil! We should sit down to a nice lunch with their leader and discuss these issues like adults.'", picture: 'isis')
+Issue.create(topic: "ISIS", description: "We can already hear the arguments unfolding and the debates haven't even started. Republicans: 'In conjunction with sharing our deep held values of A-MUR-I-CA, cheeseburgers included, we are making great progress against ISIS with every drone strike. They're the best!' Democrats: 'No no no, drone attacks are the work of the devil! We should sit down to a nice lunch with their leaders and discuss these issues like adults.'", picture: 'isis')
```

Comment:
Line is too long. [486/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.